scripts/emulator_extra.py

```
@@ -0,0 +1,3 @@
+Import("env")
```

Ruff (0.15.12)

[error] Undefined name `Import`

(F821)